src/robusta/core/playbooks/common.py

```diff
@@ -29,13 +32,15 @@ def get_resource_events_table(table_name: str, kind: str, name: str, namespace:
 
 
 def get_event_timestamp(event: Event):
-    if event.lastTimestamp:
+    if event.metadata.creationTimestamp:
```

Comment:
i think the creationTimestamp should be the last option.
The event is being created, then updated.
So the lastTimestamp is probably the most important, and should be first